### DISCUSSION TITLE
JSON Array - Spalten Auswählen

### DISCUSSION THREAD

**Discussion by flyboy70** [Zefau/ioBroker.jarvis#2858] [Q&A] [answered]
Hallo, ich verwende IOBRoker/Jarvis 3.1 (Fahrplan) und versuche ein JSON Array anzupassen und unnötige Spalten auszublenden. Der Adapter funktioniert sehr gut, aber ich möchte die Visualisierung anpassen. 

Für die "Journeys" gibts ein eigene Datapunkt, der alle DPs richtig darstellt. Ich möchte aber ein Tabelle mit nur ein paar Spalten erstellen. 

Anbei der Recordset von EIN Journey. Es kann aber 20-30 Journeys sein. 

**2\|#VN#1#ST#1734607176#PI#3#ZI#179995#TA#19#DA#201224#1S#691671#1T#1100#LS#691273#LT#1151#PU#181#RT#1#CA#Bus#ZE#62#ZB#Bus 62 #PC#6#FR#691671#FT#1100#TO#691273#TT#1151# |**  

Kann mir jemand hier der richtige Syntax geben wo ich zb EINE Spalte auswählen kann - dann kann ich weiter selbe basteln. 

![2024-12-20 11_27_38-objects - IOBroker und 20 weitere Seiten - Geschäftlich – Microsoft​ Edge](https://github.com/user-attachments/assets/a164ecb5-02cc-446c-8873-3aceb212f2e1)
![2024-12-20 11_27_10-objects - IOBroker und 20 weitere Seiten - Geschäftlich – Microsoft​ Edge](https://github.com/user-attachments/assets/aecce48d-c8e6-4b9b-861c-a47c18e7db16)

vielen Dank !

**Comment by mcuiobroker** [comment]
Bitte den Inhalt vom JSON DP

**Reply by flyboy70** [reply]
Anbei der gesamten JSON (für alle Fahrten)

[
  {
    "tripId": "2|#VN#1#ST#1734607176#PI#3#ZI#179995#TA#22#DA#201224#1S#691671#1T#1130#LS#691273#LT#1221#PU#181#RT#1#CA#Bus#ZE#62#ZB#Bus 62  #PC#6#FR#691671#FT#1130#TO#691273#TT#1221#",
    "stop": {
      "type": "stop",
      "id": "691428",
      "name": "Graz Asperngasse",
      "location": {
        "type": "location",
        "id": "691428",
        "latitude": 47.072805,
        "longitude": 15.412347
      },
      "products": {
        "nationalExpress": false,
        "national": false,
        "interregional": false,
        "regional": false,
        "suburban": false,
        "bus": true,
        "ferry": false,
        "subway": false,
        "tram": true,
        "onCall": false
      }
    },
    "when": "2024-12-20T11:51:00+01:00",
    "plannedWhen": "2024-12-20T11:48:00+01:00",
    "delay": 180,
    "platform": null,
    "plannedPlatform": null,
    "prognosisType": "prognosed",
    "direction": "Puntigam",
    "provenance": null,
    "line": {
      "type": "line",
      "id": "bus-62",
      "fahrtNr": "922439",
      "name": "Bus 62",
      "public": true,
      "adminCode": "s3800_",
      "productName": "Bus",
      "mode": "bus",
      "product": "bus",
      "operator": {
        "type": "operator",
        "id": "holding-graz-linien",
        "name": "Holding Graz Linien"
      }
    },
    "remarks": [
      {
        "type": "hint",
        "code": "OB",
        "text": "Niederflurfahrzeug"
      },
      {
        "type": "hint",
        "code": "RO",
        "text": "Rollstuhlstellplatz"
      }
    ],
    "origin": null,
    "destination": {
      "type": "stop",
      "id": "691273",
      "name": "Graz Puntigam Bahnhof (Westseite)",
      "location": {
        "type": "location",
        "id": "691273",
        "latitude": 47.030322,
        "longitude": 15.4286
      },
      "products": {
        "nationalExpress": false,
        "national": false,
        "interregional": false,
        "regional": false,
        "suburban": false,
        "bus": true,
        "ferry": false,
        "subway": false,
        "tram": true,
        "onCall": false
      }
    },
    "currentTripPosition": {
      "type": "location",
      "latitude": 47.072975,
      "longitude": 15.408194
    }
  },
  {
    "tripId": "2|#VN#1#ST#1734607176#PI#3#ZI#373142#TA#23#DA#201224#1S#691090#1T#1142#LS#691114#LT#1224#PU#181#RT#1#CA#str#ZE#1#ZB#Tram 1  #PC#9#FR#691090#FT#1142#TO#691114#TT#1224#",
    "stop": {
      "type": "stop",
      "id": "691428",
      "name": "Graz Asperngasse",
      "location": {
        "type": "location",
        "id": "691428",
        "latitude": 47.072805,
        "longitude": 15.412347
      },
      "products": {
        "nationalExpress": false,
        "national": false,
        "interregional": false,
        "regional": false,
        "suburban": false,
        "bus": true,
        "ferry": false,
        "subway": false,
        "tram": true,
        "onCall": false
      }
    },
    "when": "2024-12-20T11:51:00+01:00",
    "plannedWhen": "2024-12-20T11:49:00+01:00",
    "delay": 120,
    "platform": null,
    "plannedPlatform": null,
    "prognosisType": "prognosed",
    "direction": "Mariatrost",
    "provenance": null,
    "line": {
      "type": "line",
      "id": "tram-1",
      "fahrtNr": "900413",
      "name": "Tram 1",
      "public": true,
      "adminCode": "s3300_",
      "productName": "Tram",
      "mode": "train",
      "product": "tram",
      "operator": {
        "type": "operator",
        "id": "holding-graz-linien",
        "name": "Holding Graz Linien"
      }
    },
    "remarks": [
      {
        "type": "hint",
        "code": "OB",
        "text": "Niederflurfahrzeug"
      },
      {
        "type": "hint",
        "code": "RO",
        "text": "Rollstuhlstellplatz"
      }
    ],
    "origin": null,
    "destination": {
      "type": "stop",
      "id": "691114",
      "name": "Graz Mariatrost Tramwaymuseum",
      "location": {
        "type": "location",
        "id": "691114",
        "latitude": 47.104725,
        "longitude": 15.489268
      },
      "products": {
        "nationalExpress": false,
        "national": false,
        "interregional": false,
        "regional": false,
        "suburban": false,
        "bus": false,
        "ferry": false,
        "subway": false,
        "tram": true,
        "onCall": false
      }
    },
    "currentTripPosition": {
      "type": "location",
      "latitude": 47.070629,
      "longitude": 15.413947
    }
  },
  {
    "tripId": "2|#VN#1#ST#1734607176#PI#3#ZI#463701#TA#20#DA#201224#1S#691114#1T#1119#LS#691090#LT#1202#PU#181#RT#1#CA#str#ZE#1#ZB#Tram 1  #PC#9#FR#691114#FT#1119#TO#691090#TT#1202#",
    "stop": {
      "type": "stop",
      "id": "691540",
      "name": "Graz Hbf (Tiefgeschoß)",
      "location": {
        "type": "location",
        "id": "691540",
        "latitude": 47.071834,
        "longitude": 15.418577
      },
      "products": {
        "nationalExpress": false,
        "national": false,
        "interregional": false,
        "regional": false,
        "suburban": false,
        "bus": true,
        "ferry": false,
        "subway": false,
        "tram": true,
        "onCall": false
      }
    },
    "when": "2024-12-20T11:52:00+01:00",
    "plannedWhen": "2024-12-20T11:51:00+01:00",
    "delay": 60,
    "platform": "B",
    "plannedPlatform": "B",
    "prognosisType": "prognosed",
    "direction": "Eggenberg/UKH",
    "provenance": null,
    "line": {
      "type": "line",
      "id": "tram-1",
      "fahrtNr": "900212",
      "name": "Tram 1",
      "public": true,
      "adminCode": "s3300_",
      "productName": "Tram",
      "mode": "train",
      "product": "tram",
      "operator": {
        "type": "operator",
        "id": "holding-graz-linien",
        "name": "Holding Graz Linien"
      }
    },
    "remarks": [
      {
        "type": "hint",
        "code": "OB",
        "text": "Niederflurfahrzeug"
      },
      {
        "type": "hint",
        "code": "RO",
        "text": "Rollstuhlstellplatz"
      }
    ],
    "origin": null,
    "destination": {
      "type": "stop",
      "id": "691090",
      "name": "Graz Eggenberg Unfallkrankenhaus",
      "location": {
        "type": "location",
        "id": "691090",
        "latitude": 47.079061,
        "longitude": 15.396733
      },
      "products": {
        "nationalExpress": false,
        "national": false,
        "interregional": false,
        "regional": false,
        "suburban": false,
        "bus": true,
        "ferry": false,
        "subway": false,
        "tram": true,
        "onCall": false
      }
    },
    "currentTripPosition": {
      "type": "location",
      "latitude": 47.071834,
      "longitude": 15.418577
    }
  },
  {
    "tripId": "2|#VN#1#ST#1734607176#PI#3#ZI#415703#TA#10#DA#201224#1S#691195#1T#1123#LS#691039#LT#1154#PU#181#RT#1#CA#str#ZE#6#ZB#Tram 6  #PC#9#FR#691195#FT#1123#TO#691039#TT#1154#",
    "stop": {
      "type": "stop",
      "id": "691487",
      "name": "Graz Hbf (Daungasse)",
      "location": {
        "type": "location",
        "id": "691487",
        "latitude": 47.072661,
        "longitude": 15.413066
      },
      "products": {
        "nationalExpress": false,
        "national": false,
        "interregional": false,
        "regional": false,
        "suburban": false,
        "bus": true,
        "ferry": false,
        "subway": false,
        "tram": true,
        "onCall": false
      }
    },
    "when": "2024-12-20T11:53:00+01:00",
    "plannedWhen": "2024-12-20T11:49:00+01:00",
    "delay": 240,
    "platform": null,
    "plannedPlatform": null,
    "prognosisType": "prognosed",
    "direction": "Smart City",
    "provenance": null,
    "line": {
      "type": "line",
      "id": "tram-6",
      "fahrtNr": "908515",
      "name": "Tram 6",
      "public": true,
      "adminCode": "s3300_",
      "productName": "Tram",
      "mode": "train",
      "product": "tram",
      "operator": {
        "type": "operator",
        "id": "holding-graz-linien",
        "name": "Holding Graz Linien"
      }
    },
    "remarks": [
      {
        "type": "hint",
        "code": "OB",
        "text": "Niederflurfahrzeug"
      },
      {
        "type": "hint",
        "code": "RO",
        "text": "Rollstuhlstellplatz"
      }
    ],
    "origin": null,
    "destination": {
      "type": "stop",
      "id": "691039",
      "name": "Graz Smart City (Schleife)",
      "location": {
        "type": "location",
        "id": "691039",
        "latitude": 47.082486,
        "longitude": 15.410388
      },
      "products": {
        "nationalExpress": false,
        "national": false,
        "interregional": false,
        "regional": false,
        "suburban": false,
        "bus": true,
        "ferry": false,
        "subway": false,
        "tram": true,
        "onCall": false
      }
    },
    "currentTripPosition": {
      "type": "location",
      "latitude": 47.070629,
      "longitude": 15.413947
    }
  },
  {
    "tripId": "2|#VN#1#ST#1734607176#PI#3#ZI#180231#TA#21#DA#201224#1S#691273#1T#1118#LS#691671#LT#1208#PU#181#RT#1#CA#Bus#ZE#62#ZB#Bus 62  #PC#6#FR#691273#FT#1118#TO#691671#TT#1208#",
    "stop": {
      "type": "stop",
      "id": "691428",
      "name": "Graz Asperngasse",
      "location": {
        "type": "location",
        "id": "691428",
        "latitude": 47.072805,
        "longitude": 15.412347
      },
      "products": {
        "nationalExpress": false,
        "national": false,
        "interregional": false,
        "regional": false,
        "suburban": false,
        "bus": true,
        "ferry": false,
        "subway": false,
        "tram": true,
        "onCall": false
      }
    },
    "when": "2024-12-20T11:53:00+01:00",
    "plannedWhen": "2024-12-20T11:51:00+01:00",
    "delay": 120,
    "platform": null,
    "plannedPlatform": null,
    "prognosisType": "prognosed",
    "direction": "WKO - WIFI / tim",
    "provenance": null,
    "line": {
      "type": "line",
      "id": "bus-62",
      "fahrtNr": "922700",
      "name": "Bus 62",
      "public": true,
      "adminCode": "s3800_",
      "productName": "Bus",
      "mode": "bus",
      "product": "bus",
      "operator": {
        "type": "operator",
        "id": "holding-graz-linien",
        "name": "Holding Graz Linien"
      }
    },
    "remarks": [
      {
        "type": "hint",
        "code": "OB",
        "text": "Niederflurfahrzeug"
      },
      {
        "type": "hint",
        "code": "RO",
        "text": "Rollstuhlstellplatz"
      }
    ],
    "origin": null,
    "destination": {
      "type": "stop",
      "id": "691671",
      "name": "Graz WKO/WIFI",
      "location": {
        "type": "location",
        "id": "691671",
        "latitude": 47.088122,
        "longitude": 15.437418
      },
      "products": {
        "nationalExpress": false,
        "national": false,
        "interregional": false,
        "regional": false,
        "suburban": false,
        "bus": true,
        "ferry": false,
        "subway": false,
        "tram": false,
        "onCall": false
      }
    },
    "currentTripPosition": {
      "type": "location",
      "latitude": 47.072975,
      "longitude": 15.408194
    }
  },
  {
    "tripId": "2|#VN#1#ST#1734607176#PI#3#ZI#172733#TA#49#DA#201224#1S#691446#1T#1141#LS#691622#LT#1214#PU#181#RT#1#CA#str#ZE#7#ZB#Tram 7  #PC#9#FR#691446#FT#1141#TO#691622#TT#1214#",
    "stop": {
      "type": "stop",
      "id": "691540",
      "name": "Graz Hbf (Tiefgeschoß)",
      "location": {
        "type": "location",
        "id": "691540",
        "latitude": 47.071834,
        "longitude": 15.418577
      },
      "products": {
        "nationalExpress": false,
        "national": false,
        "interregional": false,
        "regional": false,
        "suburban": false,
        "bus": true,
        "ferry": false,
        "subway": false,
        "tram": true,
        "onCall": false
      }
    },
    "when": "2024-12-20T11:53:00+01:00",
    "plannedWhen": "2024-12-20T11:53:00+01:00",
    "delay": 0,
    "platform": "A",
    "plannedPlatform": "A",
    "prognosisType": "prognosed",
    "direction": "LKH Med Uni",
    "provenance": null,
    "line": {
      "type": "line",
      "id": "tram-7",
      "fahrtNr": "911047",
      "name": "Tram 7",
      "public": true,
      "adminCode": "s3300_",
      "productName": "Tram",
      "mode": "train",
      "product": "tram",
      "operator": {
        "type": "operator",
        "id": "holding-graz-linien",
        "name": "Holding Graz Linien"
      }
    },
    "remarks": [
      {
        "type": "hint",
        "code": "OB",
        "text": "Niederflurfahrzeug"
      },
      {
        "type": "hint",
        "code": "RO",
        "text": "Rollstuhlstellplatz"
      }
    ],
    "origin": null,
    "destination": {
      "type": "stop",
      "id": "691622",
      "name": "Graz St.Leonhard LKH Med Uni/Klinikum Nord",
      "location": {
        "type": "location",
        "id": "691622",
        "latitude": 47.082082,
        "longitude": 15.469222
      },
      "products": {
        "nationalExpress": false,
        "national": false,
        "interregional": false,
        "regional": false,
        "suburban": false,
        "bus": false,
        "ferry": false,
        "subway": false,
        "tram": true,
        "onCall": false
      }
    },
    "currentTripPosition": {
      "type": "location",
      "latitude": 47.071196,
      "longitude": 15.416132
    }
  },
  {
    "tripId": "2|#VN#1#ST#1734607176#PI#3#ZI#401443#TA#4#DA#201224#1S#691539#1T#1153#LS#696813#LT#1239#PU#181#RT#1#CA#Bus#ZE#711#ZB#Bus 711 #PC#6#FR#691539#FT#1153#TO#696813#TT#1239#",
    "stop": {
      "type": "stop",
      "id": "691539",
      "name": "Graz Hbf (Europaplatz)",
      "location": {
        "type": "location",
        "id": "691539",
        "latitude": 47.072939,
        "longitude": 15.41792
      },
      "products": {
        "nationalExpress": false,
        "national": false,
        "interregional": false,
        "regional": false,
        "suburban": false,
        "bus": true,
        "ferry": false,
        "subway": false,
        "tram": false,
        "onCall": false
      }
    },
    "when": "2024-12-20T11:53:00+01:00",
    "plannedWhen": "2024-12-20T11:53:00+01:00",
    "delay": null,
    "platform": "G",
    "plannedPlatform": "G",
    "prognosisType": null,
    "direction": "Premstätten-Tobelbad Bf via Mantscha",
    "provenance": null,
    "line": {
      "type": "line",
      "id": "bus-711",
      "fahrtNr": "902566",
      "name": "Bus 711",
      "public": true,
      "adminCode": "s94BZ9",
      "productName": "Bus",
      "mode": "bus",
      "product": "bus",
      "operator": {
        "type": "operator",
        "id": "nahreisezug",
        "name": "Nahreisezug"
      }
    },
    "remarks": [
      {
        "type": "hint",
        "code": "OB",
        "text": "Niederflurfahrzeug"
      },
      {
        "type": "hint",
        "code": "RO",
        "text": "Rollstuhlstellplatz"
      }
    ],
    "origin": null,
    "destination": {
      "type": "stop",
      "id": "696813",
      "name": "Premstätten b.Graz-Tobelbad Bahnhst (Bahnhofstraße",
      "location": {
        "type": "location",
        "id": "696813",
        "latitude": 46.979595,
        "longitude": 15.390755
      },
      "products": {
        "nationalExpress": false,
        "national": true,
        "interregional": false,
        "regional": false,
        "suburban": false,
        "bus": true,
        "ferry": false,
        "subway": false,
        "tram": false,
        "onCall": false
      }
    }
  },
  {
    "tripId": "2|#VN#1#ST#1734607176#PI#3#ZI#373142#TA#23#DA#201224#1S#691090#1T#1142#LS#691114#LT#1224#PU#181#RT#1#CA#str#ZE#1#ZB#Tram 1  #PC#9#FR#691090#FT#1142#TO#691114#TT#1224#",
    "stop": {
      "type": "stop",
      "id": "691540",
      "name": "Graz Hbf (Tiefgeschoß)",
      "location": {
        "type": "location",
        "id": "691540",
        "latitude": 47.071834,
        "longitude": 15.418577
      },
      "products": {
        "nationalExpress": false,
        "national": false,
        "interregional": false,
        "regional": false,
        "suburban": false,
        "bus": true,
        "ferry": false,
        "subway": false,
        "tram": true,
        "onCall": false
      }
    },
    "when": "2024-12-20T11:54:00+01:00",
    "plannedWhen": "2024-12-20T11:52:00+01:00",
    "delay": 120,
    "platform": "A",
    "plannedPlatform": "A",
    "prognosisType": "prognosed",
    "direction": "Mariatrost",
    "provenance": null,
    "line": {
      "type": "line",
      "id": "tram-1",
      "fahrtNr": "900413",
      "name": "Tram 1",
      "public": true,
      "adminCode": "s3300_",
      "productName": "Tram",
      "mode": "train",
      "product": "tram",
      "operator": {
        "type": "operator",
        "id": "holding-graz-linien",
        "name": "Holding Graz Linien"
      }
    },
    "remarks": [
      {
        "type": "hint",
        "code": "OB",
        "text": "Niederflurfahrzeug"
      },
      {
        "type": "hint",
        "code": "RO",
        "text": "Rollstuhlstellplatz"
      }
    ],
    "origin": null,
    "destination": {
      "type": "stop",
      "id": "691114",
      "name": "Graz Mariatrost Tramwaymuseum",
      "location": {
        "type": "location",
        "id": "691114",
        "latitude": 47.104725,
        "longitude": 15.489268
      },
      "products": {
        "nationalExpress": false,
        "national": false,
        "interregional": false,
        "regional": false,
        "suburban": false,
        "bus": false,
        "ferry": false,
        "subway": false,
        "tram": true,
        "onCall": false
      }
    },
    "currentTripPosition": {
      "type": "location",
      "latitude": 47.070629,
      "longitude": 15.413947
    }
  },
  {
    "tripId": "2|#VN#1#ST#1734607176#PI#3#ZI#408589#TA#16#DA#201224#1S#691449#1T#1129#LS#691663#LT#1201#PU#181#RT#1#CA#str#ZE#4#ZB#Tram 4  #PC#9#FR#691449#FT#1129#TO#691663#TT#1201#",
    "stop": {
      "type": "stop",
      "id": "691540",
      "name": "Graz Hbf (Tiefgeschoß)",
      "location": {
        "type": "location",
        "id": "691540",
        "latitude": 47.071834,
        "longitude": 15.418577
      },
      "products": {
        "nationalExpress": false,
        "national": false,
        "interregional": false,
        "regional": false,
        "suburban": false,
        "bus": true,
        "ferry": false,
        "subway": false,
        "tram": true,
        "onCall": false
      }
    },
    "when": "2024-12-20T11:54:00+01:00",
    "plannedWhen": "2024-12-20T11:52:00+01:00",
    "delay": 120,
    "platform": "B",
    "plannedPlatform": "B",
    "prognosisType": "prognosed",
    "direction": "Reininghaus",
    "provenance": null,
    "line": {
      "type": "line",
      "id": "tram-4",
      "fahrtNr": "904320",
      "name": "Tram 4",
      "public": true,
      "adminCode": "s3300_",
      "productName": "Tram",
      "mode": "train",
      "product": "tram",
      "operator": {
        "type": "operator",
        "id": "holding-graz-linien",
        "name": "Holding Graz Linien"
      }
    },
    "remarks": [
      {
        "type": "hint",
        "code": "OB",
        "text": "Niederflurfahrzeug"
      },
      {
        "type": "hint",
        "code": "RO",
        "text": "Rollstuhlstellplatz"
      }
    ],
    "origin": null,
    "destination": {
      "type": "stop",
      "id": "691663",
      "name": "Graz Reininghaus (Schleife)",
      "location": {
        "type": "location",
        "id": "691663",
        "latitude": 47.056804,
        "longitude": 15.410217
      },
      "products": {
        "nationalExpress": false,
        "national": false,
        "interregional": false,
        "regional": false,
        "suburban": false,
        "bus": false,
        "ferry": false,
        "subway": false,
        "tram": true,
        "onCall": false
      }
    },
    "currentTripPosition": {
      "type": "location",
      "latitude": 47.071232,
      "longitude": 15.424635
    }
  },
  {
    "tripId": "2|#VN#1#ST#1734607176#PI#3#ZI#416142#TA#17#DA#201224#1S#691039#1T#1145#LS#691195#LT#1216#PU#181#RT#1#CA#str#ZE#6#ZB#Tram 6  #PC#9#FR#691039#FT#1145#TO#691195#TT#1216#",
    "stop": {
      "type": "stop",
      "id": "691540",
      "name": "Graz Hbf (Tiefgeschoß)",
      "location": {
        "type": "location",
        "id": "691540",
        "latitude": 47.071834,
        "longitude": 15.418577
      },
      "products": {
        "nationalExpress": false,
        "national": false,
        "interregional": false,
        "regional": false,
        "suburban": false,
        "bus": true,
        "ferry": false,
        "subway": false,
        "tram": true,
        "onCall": false
      }
    },
    "when": "2024-12-20T11:54:00+01:00",
    "plannedWhen": "2024-12-20T11:53:00+01:00",
    "delay": 60,
    "platform": "A",
    "plannedPlatform": "A",
    "prognosisType": "prognosed",
    "direction": "St.Peter",
    "provenance": null,
    "line": {
      "type": "line",
      "id": "tram-6",
      "fahrtNr": "908817",
      "name": "Tram 6",
      "public": true,
      "adminCode": "s3300_",
      "productName": "Tram",
      "mode": "train",
      "product": "tram",
      "operator": {
        "type": "operator",
        "id": "holding-graz-linien",
        "name": "Holding Graz Linien"
      }
    },
    "remarks": [
      {
        "type": "hint",
        "code": "OB",
        "text": "Niederflurfahrzeug"
      },
      {
        "type": "hint",
        "code": "RO",
        "text": "Rollstuhlstellplatz"
      }
    ],
    "origin": null,
    "destination": {
      "type": "stop",
      "id": "691195",
      "name": "Graz St.Peter (Schleife)",
      "location": {
        "type": "location",
        "id": "691195",
        "latitude": 47.058647,
        "longitude": 15.480566
      },
      "products": {
        "nationalExpress": false,
        "national": false,
        "interregional": false,
        "regional": false,
        "suburban": false,
        "bus": true,
        "ferry": false,
        "subway": false,
        "tram": true,
        "onCall": false
      }
    },
    "currentTripPosition": {
      "type": "location",
      "latitude": 47.070629,
      "longitude": 15.413947
    }
  },
  {
    "tripId": "2|#VN#1#ST#1734607176#PI#3#ZI#415701#TA#11#DA#201224#1S#691195#1T#1130#LS#691039#LT#1201#PU#181#RT#1#CA#str#ZE#6#ZB#Tram 6  #PC#9#FR#691195#FT#1130#TO#691039#TT#1201#",
    "stop": {
      "type": "stop",
      "id": "691540",
      "name": "Graz Hbf (Tiefgeschoß)",
      "location": {
        "type": "location",
        "id": "691540",
        "latitude": 47.071834,
        "longitude": 15.418577
      },
      "products": {
        "nationalExpress": false,
        "national": false,
        "interregional": false,
        "regional": false,
        "suburban": false,
        "bus": true,
        "ferry": false,
        "subway": false,
        "tram": true,
        "onCall": false
      }
    },
    "when": "2024-12-20T11:55:00+01:00",
    "plannedWhen": "2024-12-20T11:53:00+01:00",
    "delay": 120,
    "platform": "B",
    "plannedPlatform": "B",
    "prognosisType": "prognosed",
    "direction": "Smart City",
    "provenance": null,
    "line": {
      "type": "line",
      "id": "tram-6",
      "fahrtNr": "908516",
      "name": "Tram 6",
      "public": true,
      "adminCode": "s3300_",
      "productName": "Tram",
      "mode": "train",
      "product": "tram",
      "operator": {
        "type": "operator",
        "id": "holding-graz-linien",
        "name": "Holding Graz Linien"
      }
    },
    "remarks": [
      {
        "type": "hint",
        "code": "OB",
        "text": "Niederflurfahrzeug"
      },
      {
        "type": "hint",
        "code": "RO",
        "text": "Rollstuhlstellplatz"
      }
    ],
    "origin": null,
    "destination": {
      "type": "stop",
      "id": "691039",
      "name": "Graz Smart City (Schleife)",
      "location": {
        "type": "location",
        "id": "691039",
        "latitude": 47.082486,
        "longitude": 15.410388
      },
      "products": {
        "nationalExpress": false,
        "national": false,
        "interregional": false,
        "regional": false,
        "suburban": false,
        "bus": true,
        "ferry": false,
        "subway": false,
        "tram": true,
        "onCall": false
      }
    },
    "currentTripPosition": {
      "type": "location",
      "latitude": 47.071079,
      "longitude": 15.428312
    }
  }
]

**Reply by mcuiobroker** [reply]
Was soll jetzt vom Json wie in der Tabelle stehen?

**Reply by flyboy70** [reply]
When | Direction | Line.Name | Delay

zB
10:58 - Wien Hbf - X31 - 5mins

Danke Dir !!

**Comment by mcuiobroker** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/aebf5c7c-3376-4bc7-9e51-12ec91a907d9)
Modul Konfig
```
[{"field":"when"},{"field":"direction"},{"field":"line.name"},{"field":"delay"}]
```

![image](https://github.com/user-attachments/assets/06caa882-e909-4b82-ad6a-6d3e50d6dfeb)

```
/* jDate() ab v3.2.0, sonst date() */
[{"field":"when","format":"val => val = jDate(val,'dd.MM.yyy HH:mm')"},{"field":"direction"},{"field":"line.name"},{"field":"delay"}]
```

**Reply by flyboy70** [reply]
Bist genial . DANKE DIR !!
